pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "astro-image-display-api"
+dynamic = ["version"]
+description = 'Astronomical image display API definition for cross-backend interoperability'
+readme = "README.md"
+requires-python = ">=3.11"
+license = "BSD-3-Clause"
+keywords = []
+authors = [
+  { name = "Matt Craig", email = "mattwcraig@gmail.com" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = []
+
+[project.urls]
+Documentation = "https://github.com/astropy/astro-image-display-api#readme"
+Issues = "https://github.com/astropy/astro-image-display-api/issues"
+Source = "https://github.com/astropy/astro-image-display-api"
+
+[tool.hatch.version]
+path = "src/astro_image_display_api/__about__.py"
+
+[tool.coverage.run]
+source_pkgs = ["astro_image_display_api", "tests"]
+branch = true
+parallel = true
+omit = [
+  "src/astro_image_display_api/__about__.py",
+]
+
+[tool.coverage.paths]
+astro_image_display_api = ["src/astro_image_display_api", "*/astro-image-display-api/src/astro_image_display_api"]
+tests = ["tests", "*/astro-image-display-api/tests"]
```

Comment:
If you want to do the src + tests layout, why even  have `astro-image-display-api/tests`?

Reply:
Oops, that is an accident.